studio/components/interfaces/Organization/TeamSettings/TeamSettings.tsx

```diff
@@ -24,6 +24,11 @@ const TeamSettings = observer(() => {
   const [isLeaving, setIsLeaving] = useState(false)
   const canAddMembers = rolesAddable.length > 0
 
+  const canLeave = (isOwner && ui.isOwnerAndCanLeaveOrg(members)) || !isOwner
```

Comment:
`(isOwner && ui.isOwnerAndCanLeaveOrg(members))`

i think the `isOwner` check here is unnecessary right? can just be
`const canLeave = (ui.isOwnerAndCanLeaveOrg(members)) || !isOwner`